waveorder/models/isotropic_thin_3d.py

```diff
@@ -0,0 +1,217 @@
+import torch
+import numpy as np
+from waveorder import optics, util
+
+
+def generate_test_phantom(
+    yx_shape,
+    yx_pixel_size,
+    wavelength_illumination,
+    index_of_refraction_media,
+    index_of_refraction_sample,
+    sphere_radius,
+):
+    sphere, _, _ = util.generate_sphere_target(
+        (3,) + yx_shape,
+        yx_pixel_size,
+        z_pixel_size=1.0,
+        radius=sphere_radius,
+        blur_size=2 * yx_pixel_size,
+    )
+    yx_phase = (
+        sphere[1]
+        * (index_of_refraction_sample - index_of_refraction_media)
+        * 0.1
+        / wavelength_illumination
+    )  # phase in radians
+
+    yx_absorption = 0.99 * sphere[1]
+
+    return yx_absorption, yx_phase
+
+
+def calculate_transfer_function(
+    yx_shape,
+    yx_pixel_size,
+    z_position_list,
```

Comment:
I didn't catch this in #131, but we need a better name and/or more documentation to indicate that this is a `torch.Tensor`, not a `list`.

Reply:
(Looking more into the history it used to be a `list`.)

Reply:
Great catch @ziw-liu, and your test case helped me sort this out.

Initially, I named it `_list` because I wanted it to be a `list`. But then your tests were failing and I "fixed" it by passing a tensor, which was a false fix.

Now I'm passing in a list in the test case, and I fixed the issue.